SConstruct

```diff
@@ -891,21 +891,24 @@ if env.msvc and not methods.using_clang(env):  # MSVC
         env.AppendUnique(LINKFLAGS=["/WX"])
 
 else:  # GCC, Clang
-    common_warnings = ["-Wenum-conversion"]
-
+    common_warnings = []
     if methods.using_gcc(env):
         common_warnings += ["-Wshadow", "-Wno-misleading-indentation"]
         if cc_version_major < 11:
             # Regression in GCC 9/10, spams so much in our variadic templates
             # that we need to outright disable it.
             common_warnings += ["-Wno-type-limits"]
```

Comment:
For the record, I tested with GCC 9 on Ubuntu 20.04 to try to reproduce these with `scons warnings=all`, but didn't trigger them. We might have fixed it somehow by refactoring our templates over time, or maybe it got fixed in later GCC 9/10 patch releases include the current version as of Ubuntu 20.04 EOL.

So we _could_ in theory try to remove this, but I haven't done a thorough testing with different optimization flags and GCC 9/10 versions, and overall we don't really care about those old versions so there's no need to micro-optimize here. Sooner or later we'll drop support for these GCC versions (likely when moving to C++20).

Reply:
I say we leave it; no need to fuss over these older versions if we're already covering our bases